### ISSUE TITLE
allKeys.flatMap is not a function

### ISSUE THREAD

**Issue by vriad** [iamvishnusankar/next-sitemap#22]
Getting this error when I try to build a sitemap.

I'm using Next.js 9.3 with the default `next-sitemap.js`:

```
module.exports = {
  siteUrl: 'https://example.com',
  generateRobotsTxt: true, // (optional)
  // ...other options
}
```

Looks like the offending code is [here](https://github.com/iamvishnusankar/next-sitemap/blob/0a4debdeede39ec023d7a4606479dc765a354271/packages/next-sitemap/src/url/create-url-set/index.ts) but I'm not sure what's going wrong exactly. 

![Screen Shot 2020-08-23 at 5 16 14 PM](https://user-images.githubusercontent.com/3084745/90992509-1a10fd00-e565-11ea-977e-fa20e19d79b8.png)

I'm also attaching a ZIP of some of the manifests which may help debug this. I tried for a while but couldn't figure it out.

[manifests.zip](https://github.com/iamvishnusankar/next-sitemap/files/5115239/manifests.zip)

**Comment by iamvishnusankar** [comment]
@vriad Thanks for reporting this. I tried building using your manifest files and it's working fine on my local machine. 

I tried running a matrix test and looks like the build is failing with node version < 11. Check the build stats here: https://github.com/iamvishnusankar/next-sitemap/runs/1019782488?check_suite_focus=true#step:5:10077

 What's the node version you're using? Array.flatMap is supported on node >=11 only.
